src/kboot.c

```diff
@@ -349,6 +349,52 @@ static int dt_set_serial_number(void)
     return 0;
 }
 
+static int dt_set_smbios(void)
+{
+    int fdt_root = fdt_path_offset(dt, "/");
+    int len, node, smbios_node;
+
+    if (fdt_root < 0)
+        bail("FDT: could not open a handle to FDT root.\n");
+
+    node = fdt_add_subnode(dt, fdt_root, "smbios");
+    if (node < 0)
+        bail("FDT: failed to add node 'smbios' to  '/'\n");
+    fdt_setprop_string(dt, node, "compatible", "u-boot,sysinfo-smbios");
+
+    smbios_node = fdt_add_subnode(dt, node, "smbios");
+    if (smbios_node < 0)
+        bail("FDT: failed to add node 'smbios' to  '/smbios'\n");
+
+    node = fdt_add_subnode(dt, smbios_node, "system");
+    if (node < 0)
+        bail("FDT: failed to add node 'system' to  '/smbios'\n");
+    fdt_setprop_string(dt, node, "manufacturer", "Apple");
```

Comment:
Please use `/manufacturer` from the ADT. Value is "Apple Inc." which matches `dmidecode` on Intel Macs. same for baseboard and chassis

Reply:
Sure